### PR TITLE
feat: mark junk mail automatically as read and unimportant

### LINKED ISSUE #9710: Automatically mark mail as read and remove important tag  when manually marked as junk
### Is your feature request related to a problem? Please describe.

I am just moving to NC mail as my desktop mail client and I noticed that when marking a mail as spam/junk, it is moved to the junk folder but it remains unread and a potential important classification is not removed. That needs to be done manually, increasing the workload for high volume accounts.

### Describe the solution you'd like

NC mail should automatically mark a mail as read and remove a potential important classification when a mail is manually marked as junk.

### Describe alternatives you've considered

The alternative is to do it manually, which - as described - increases the workload on high volume accounts.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by PeterJGDe**
fixes #9710

### CODE REVIEW

File: src/components/Envelope.vue [hunk 1]
```diff
@@ -727,6 +727,15 @@ export default {
 		async onToggleJunk() {
 			const removeEnvelope = await this.$store.dispatch('moveEnvelopeToJunk', this.data)
 
+			if (this.isImportant) {
+				this.$store.dispatch('toggleEnvelopeImportant', this.data)
```
Comment (ChristophWurst) [inline src/components/Envelope.vue:731]:
```suggestion
				await this.$store.dispatch('toggleEnvelopeImportant', this.data)
```

File: src/components/Envelope.vue [hunk 2]
```diff
@@ -727,6 +727,15 @@ export default {
...
+			}
+
+			if (!this.data.flags.seen) {
+			console.log('toggleEnvelopeSeen')
```
Comment (ChristophWurst) [inline src/components/Envelope.vue:735]: remove this line please

File: src/components/Envelope.vue [hunk 3]
```diff
@@ -727,6 +727,15 @@ export default {
...
+				this.$store.dispatch('toggleEnvelopeSeen', { envelope: this.data })
```
Comment (ChristophWurst) [inline src/components/Envelope.vue:736]:
```suggestion
				await this.$store.dispatch('toggleEnvelopeSeen', { envelope: this.data })
```

**Review by ChristophWurst — APPROVED** [review]

**Comment by sgofferj** [comment]
Fantastic! Thank you for your work!

**Pull request merged**